docs/examples/en/controls/TrackballControls.html

```diff
@@ -80,7 +80,7 @@ <h3>[property:Array keys]</h3>
 				<li>When the second defined key is pressed, all mouse interactions (left, middle, right) performs zooming.</li>
 				<li>When the third defined key is pressed, all mouse interactions (left, middle, right) performs panning.</li>
 			</ul>
-			Default is *65, 83, 68* which represents A, S, D.
+			Default is *'KeyA', 'KeyS', 'KeyD'* which represents A, S, D.
```

Comment:
It seems `'KeyA', 'KeyS', 'KeyD'` is now not bold anymore. Meaning the stuff enclosed by two `*` signs. Probably because the parse can't process `'` signs. I suggest you remove them and try again. 
```js
*KeyA, KeyS, KeyD* 
```